ExodusMovement/react-native-webview
# chore: remove macos

## Summary
Removes macos

I was able to validate the build (from the included examples) on both iOS and Android 

## Test Plan
- make sure `yarn ios` works
- make sure `yarn android` works
- both are included examples in rn webview
- preconditions are that you have already established a dev env for RN
